src/app/features/course/course.component.html

```diff
@@ -3,7 +3,7 @@
         <mat-form-field class="example-full-width" appearance="fill">
```

Comment:
![image](https://user-images.githubusercontent.com/83765331/118826238-f98f8c80-b8c3-11eb-9580-44e6954c3360.png)
all fields are filled, but form is not valid, why?